frontend/src/NavigationPath.tsx

```diff
@@ -154,6 +154,7 @@ export enum NavigationPath {
   policyDetailsResults = '/multicloud/governance/policies/details/:namespace/:name/results',
   policyDetailsHistory = '/multicloud/governance/policies/details/:namespace/:name/status/:clusterName/templates/:templateName/history',
   policyTemplateDetails = '/multicloud/governance/policies/details/:namespace/:name/template/:clusterName/:apiGroup?/:apiVersion/:kind/:templateName',
+  policyTemplateYaml = '/multicloud/governance/policies/details/:namespace/:name/template/:clusterName/:apiGroup?/:apiVersion/:kind/:templateName/yaml',
```

Comment:
Could you make the "yaml" an optional URL parameter instead and just have the one route? This way, your code of if ends with `/yaml` won't get triggered if the template name is `yaml` for some odd reason.

Reply:
``location.pathname.endsWith('/yaml')` `This check "/yaml" also policyDetailsHistory, policyDetailsResults this way.